earthfile2llb/converter.go

```diff
@@ -1691,6 +1692,19 @@ func (c *Converter) internalRun(ctx context.Context, opts ConvertRunOpts) (pllb.
 	}
 	if !opts.Locally {
 		// Debugger.
+		err := c.opt.LLBCaps.Supports(solverpb.CapExecMountSock)
+		if err != nil {
+			_, isCapError := err.(*apicaps.CapError)
+			if !isCapError {
+				c.opt.Console.Warnf("failed to check LLBCaps for CapExecMountSock: %v", err) // keep going
+			} else if opts.Interactive {
+				// only display warning when user supplies --interactive to an older buildkit version
+				c.opt.Console.Warnf("interactive debugger requires a newer version of buildkit: %v", err)
+			}
+		} else {
+			runOpts = append(runOpts, llb.SocketTarget("earthly_interactive", "/var/run/earthly_interactive", 0666, 0, 0))
```

Comment:
Presumably, this creates `/var/run/` under the hood if it doesn't exist yet, right?

Reply:
good question! I took it for granted, but here's a quick manual test:

busybox doesn't have `/var/run`:
```
$ docker run --rm busybox:latest ls -la /var/run
ls: /var/run: No such file or directory
```

but buildkit will create it:

```
$ ./build/linux/amd64/earthly -i ~/test/socket-server+novarrun 
 1. Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)

 2. Build 🔧
————————————————————————————————————————————————————————————————————————————————

      busybox:latest | --> Load metadata linux/amd64
/h/a/t/socket-server+novarrun | --> FROM busybox:latest
/h/a/t/socket-server+novarrun | [██████████] 100% resolve docker.io/library/busybox:latest@sha256:20142e89dab967c01765b0aea3be4cec3a5957cc330f061e5503ef6168ae6613
/h/a/t/socket-server+novarrun | --> RUN --no-cache false
/h/a/t/socket-server+novarrun | Entering interactive debugger
/h/a/t/socket-server+novarrun | Command /bin/sh -c false failed with exit code 1
/h/a/t/socket-server+novarrun | 

/ # ls -la /var/run
total 8
drwxr-xr-x    2 root     root          4096 Sep 13 00:00 .
drwxr-xr-x    1 root     root          4096 Sep 13 00:00 ..
srw-rw-rw-    1 root     root             0 Sep 13 00:00 earthly_interactive
```